api/python/provisioner/commands/set_swupgrade_repo.py

```diff
@@ -32,8 +32,74 @@
 
 class SetSWUpgradeRepo(SetSWUpdateRepo):
 
-    _REPO_DEST = PRVSNR_USER_FILES_SWUPGRADE_REPOS_DIR
     _BASE_DIR_PILLAR = "release/upgrade/base_dir"
+    _INSTALL_TYPE_PILLAR_KEY = "release/install_type"
+
+    def _configure_install_type(self, install_type: str) -> PillarKey:
+        """
+        Configure the installation type for SW upgrade
+
+        Parameters
+        ----------
+        install_type: str
+            value for `install_type` pillar key
+
+        Returns
+        -------
+        PillarKey
+            PillarKey for the further `install_type` pillar update
+
+        """
+        install_type_path = PillarKey(self._INSTALL_TYPE_PILLAR_KEY)
+        return inputs.PillarInputBase.from_args(
+                                        keypath=install_type_path,
+                                        value=install_type)
+
+    def _get_current_install_type(self) -> str:
+        """
+        Get current install type for the local minion
+
+        Returns
+        -------
+        str
+            return `release/install_type` value for backup
+
+        """
+        pillar_path = PillarKey(self._INSTALL_TYPE_PILLAR_KEY)
+        pillar = PillarResolver(local_minion_id()).get([pillar_path])
+
+        for target, _pillar in pillar.items():
```

Comment:
we expect only one target (local) here, looping looks as an overhead

Reply:
This code is no more used for SW Upgrade. Deleted this method